deployment/deployment.json

```diff
@@ -3692,6 +3692,22 @@
           "prepare:constants": false
         }
       }
+    },
+    "tornado-cash": {
+      "tornado-cash": {
+        "mainnet": {
+          "template": "tornado-cash.template.yaml",
+          "messari": "messari/tornado-cash-mainnet",
+          "dhruv-chauhan": "dhruv-chauhan/tornado-cash-mainnet",
+          "deploy-on-merge": true
+        },
+        "bsc": {
+          "template": "tornado-cash.template.yaml",
+          "messari": "messari/tornado-cash-bsc",
+          "dhruv-chauhan": "dhruv-chauhan/tornado-cash-bsc",
+          "deploy-on-merge": true
+        }
+      }
```

Comment:
Take a look at the new deployment.json schema. It got updated recently

Reply:
updated :+1: